test/verify/check-session

```diff
@@ -71,9 +71,8 @@ class TestSession(MachineCase):
         wait_session(True)
 
         # Kill session from the inside
-        b.switch_to_top()
-        b.wait_visible("#content")
-        b.wait_visible("#hosts-sel")
+        b.logout()
```

Comment:
It seems like the original intention of the test is something else than what you end up testing here, no?

Reply:
Maybe, but what would it be? The users page has a "terminate session" button, but you can't use that for the currently running session. Maybe it was relying on the test to automatically clean up sessions, but as there were no assertions at the end, the whole paragraph would be pointless. I could alternatively just completely remove it, but making sure that a logout properly cleans up the session seems to make sense?

Reply:
Ya.  I actually agree with this assessment on the second read.  I have no idea what it was trying to accomplish before.